src/v/cluster/health_monitor_types.cc

```diff
@@ -263,4 +263,61 @@ std::ostream& operator<<(std::ostream& o, const get_cluster_health_reply& r) {
     return o;
 }
 
+namespace {
+template<typename Range>
+void print_range(std::ostream& o, const Range& range) {
+    if (range.size() > 0) {
+        auto it = range.begin();
+        auto end = range.end();
+        fmt::print(o, "{}", *it);
+        ++it;
+        for (; it != end; ++it) {
+            fmt::print(o, ", {}", *it);
+        }
+    }
+}
+} // namespace
+
+std::ostream&
+operator<<(std::ostream& o, const columnar_node_health_report& report) {
+    fmt::print(
+      o,
+      "{{id: {}, local_state: {}, drain_status: {}, topics: [",
+      report.id,
+      report.local_state,
+      report.drain_status);
+    print_range(o, report.topics);
```

Comment:
nit: curious why fmt::join() won't work here.

Reply:
this is from the cstore version, it was included by accident. removing